server/api/controllers/message.controller.js

```diff
@@ -27,6 +27,13 @@ module.exports = function MessageController(gladys) {
       created_at: req.body.created_at || new Date(),
     };
     gladys.event.emit(EVENTS.MESSAGE.NEW, messageToSend);
+    // We emit message.sent to synchronize all discussions tabs in Front
+    messageToSend.id = req.body.id;
```

Comment:
Why do you edit the objet here? It'll affect the object sent previously as well (in JS, parameters are passed by reference when it's an object not by copy)

Reply:
I need to create a copy object to add the id OR add the id only in the event ?

```
    gladys.event.emit(EVENTS.WEBSOCKET.SEND, {
      type: WEBSOCKET_MESSAGE_TYPES.MESSAGE.SENT,
      payload: messageToSend,
      userId: req.user.id,
      messageId: req.body.id, 
    });
```

Reply:
I don't know, you added the id :p